ws-nextjs-app/pages/[service]/send/[id]/FormField/EmailInput.tsx

```diff
@@ -23,9 +24,12 @@ export default ({
         type="email"
         value={value as string}
         onChange={e => handleChange(e.target.name, e.target.value)}
-        aria-invalid={!isValid}
-        aria-required={required}
-        aria-describedby={describedBy}
+        {...(submitAttemptCount > 0 &&
+          wasInvalid && { 'aria-invalid': !isValid })}
+        {...(submitAttemptCount > 0 &&
+          required && { 'aria-required': required })}
+        {...(submitAttemptCount > 0 &&
+          !isValid && { 'aria-describedby': describedBy })}
```

Comment:
Would something like this work in these cases that depend on `submitAttemptCount` for setting each of the values:
```
        {...(submitAttemptCount > 0 && {
          ...(wasInvalid && { 'aria-invalid': !isValid }),
          ...(required && { 'aria-required': required }),
          ...(!isValid && { 'aria-describedby': describedBy }),
        })}
```